packages/zone/src/heap.js

```diff
@@ -38,13 +35,15 @@ const detachedHeapStores = Far('heapStores', {
  */
 export const makeHeapZone = (baseLabel = 'heapZone') => {
   const { makeOnce, wrapProvider } = makeOnceKit(baseLabel, detachedHeapStores);
-  return Far('heapZone', {
+
+  const subZoneProvider = (label, _options) =>
+    makeHeapZone(`${baseLabel}.${label}`);
+
+  return harden({
     exo: wrapProvider(makeExo, keys.exo),
     exoClass: wrapProvider(defineExoClass, keys.exoClass),
     exoClassKit: wrapProvider(defineExoClassKit, keys.exoClassKit),
-    subZone: (label, _options) => {
-      return makeOnce(label, () => makeHeapZone(`${baseLabel}.${label}`));
-    },
+    subZone: wrapProvider(subZoneProvider),
```

Comment:
This works because `wrapProvider` does not enforce that the provided thing is storable, whereas `makeOnce` necessarily does.